service-front/assets/js/moj/moj.modules/moj.fees.js

```diff
@@ -1,24 +0,0 @@
-// Fees module for LPA
-// Dependencies: moj, jQuery
-
-(function () {
-  'use strict';
-
-  moj.Modules.Fees = {
-
-    init: function () {
-      this.labelSubtextDisplay();
-    },
-
-    labelSubtextDisplay: function(){
-      $('input[name=reductionOptions]').change(function(){
-        $('.revised-fee').addClass('hidden');
```

Comment:
None of these selectors appear in the PHP code or Twig templates, so this function does nothing.